.github/workflows/main.yml

```diff
@@ -218,4 +218,4 @@ jobs:
       - uses: actions/setup-node@v3
         with:
           node-version: 18
-      - run: cargo test -p cargo-risczero -F docker
+      - run: cargo test -p risc0-build
```

Comment:
This might be a mistake. I think we need to continue to run tests for `cargo-risczero`. You can just drop the `-F docker`. Also, `risc0-build` should already be getting tested by `cargo test` above which is workspace-wide.

Reply:
Thanks for catching this. So I moved everything from cargo-risczero to risc0-build and that's why I changed it. I decided to keep the -F docker in my most recent change. Do you think we should still run the test from cargo-risczero? If so, I'm totally fine moving the test from risc0-build to cargo-risczero